geekdanitek/questioner-react
# #166056179 User should see a landing page

#### Description
This PR enables the user to see a landing page

#### Type of change
- Non-breaking change

#### How Has This Been Tested?
- clone this branch `story-166056179-landing-page`
- cd in the application folder
- run `npm start:dev`
- go to `/` to see the landing page

#### Checklist:

- [x] Create UI for landing page 

#### Story (PT)
[#166056179](https://www.pivotaltracker.com/n/projects/2333428/stories/166056179)
